src/Picasso_ParticleInit.hpp

```diff
@@ -47,6 +47,7 @@ void filterEmpties( const ExecutionSpace& exec_space,
         Kokkos::ViewAllocateWithoutInitializing( "empties" ),
         std::min( num_particles - local_num_create, local_num_create ) );
     Kokkos::parallel_scan(
+        "find_empty_particles",
```

Comment:
Consider prefixing with something like `Picasso::`

Reply:
I agree with this

Reply:
Fixed